### ISSUE TITLE
User zooming and scaling 

### ISSUE THREAD

**Issue by aitchiss** [code-newbies/codeland#49]
At the moment, the site restricts users from zooming and scaling content on their mobile devices, due to:

```html
<meta content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no" name="viewport">
```

We should update this to enable user zooming, as this is a core accessibility feature - [see the MDN docs for more details and explanation](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag).

I also noticed that on mobile devices, there are some issues currently with how the content is displayed, with users needing to scroll horizontally to find the menu. For both accessibility and usability reasons, horizontal scrolling should be avoided and we should make sure the content fits the max width of the device.

The below was recorded on an Android Pixel 5:

https://user-images.githubusercontent.com/20773163/124901438-ef436200-dfd9-11eb-9bf3-82caf3164daf.mp4

**Comment by danieltott** [comment]
![i'm on it](https://media.tenor.co/images/022a3e82bcffeae373106a9d3a3a5a97/tenor.gif)
